scripts/bundle-generation/PLATFORM-AWARE-CHANGES.py

```
@@ -0,0 +1,434 @@
+    logging.debug(f"Destination path: {destination_chart_path}")
+
+    # Remove existing destination if it exists
+    if os.path.exists(destination_chart_path):
+        logging.warning(f"Destination chart path already exists. Removing: {destination_chart_path}")
+        shutil.rmtree(destination_chart_path)
+
+    # Copy the entire chart directory
+    try:
+        shutil.copytree(chart_source_path, destination_chart_path)
+        logging.info(f"Copied chart to: {destination_chart_path}")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)